### PR TITLE
Feature/zero state css

### PR DESCRIPTION
**Pull request by daomeow**
## Is this a fix or a feature?
* Fix
## What is the change?
* New activity and past activity match comp more
## What does it fix?
* Styling
## Where should the reviewer start?
* Line 20 
## How should this be tested? 
* GH Pages

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -11,41 +13,60 @@ body {
...
 
 .boxes {
-  background-color: #46424D;
+  background-color: #46424D;          
   font-family: 'Montserrat', sans-serif;
   color: white;
-  padding-top: 2%;
-  padding-left: 12%;
-  margin-left: 1%;
-  top: -440px;
```
Comment (percworld) [inline style.css:33]: try vh instead of px

File: style.css [hunk 2]
```diff
@@ -11,41 +13,60 @@ body {
...
+  height: 600px;
+  padding: 2% 0 0 8%;
+  margin: 0 38% 0 1%;
+  top: -930px;
```
Comment (percworld) [inline style.css:38]: again here?

File: style.css [hunk 3]
```diff
@@ -11,41 +13,60 @@ body {
...
   background-color: #46424D;
   border-color: white;
   color: white;
+  height:100px;
   padding: 25px 50px;
   margin-left 1%;
 }
 
-.active {
+// Buttons
```
Comment (percworld) [inline style.css:51]: /* */ in CSS

File: style.css (hunk 1 above)
Comment (percworld) [reply style.css:33]: margin-top:

File: style.css [hunk 4]
```diff
@@ -11,41 +13,60 @@ body {
...
   position: relative;
 }
 
 .btn-category {
   background-color: #46424D;
   border-color: white;
   color: white;
+  height:100px;
   padding: 25px 50px;
   margin-left 1%;
```
Comment (percworld) [inline style.css:48]: needs semicolon

**Review by percworld — APPROVED** [review]
Try using vh for height and vw for width instead of pixels-

**Review by bhohnco — APPROVED** [review]

**Pull request merged**